specs/Modal.spec.js

```diff
@@ -234,25 +244,42 @@ describe('Modal', () => {
     const modal = renderModal({ isOpen: true });
     expect(modal.portal.content.style.position).toEqual(newStyle);
     Modal.defaultStyles.content.position = previousStyle;
+    unmountModal();
+  });
+
+  it('should remove class from body when no modals opened', () => {
+    const expectedBodyClass = 'ReactModal__Body--open';
+    const classList = document.body.classList;
+    renderModal({ isOpen: true });
+    renderModal({ isOpen: true });
+    expect(classList.contains(expectedBodyClass)).toBe(true);
+    unmountModal();
+    expect(classList.contains(expectedBodyClass)).toBe(false);
```

Comment:
Ahh you're right I reversed that test inadvertently 😿 

So the design decision is to allow for modal stacking then? I naively assumed from both an accessibility and UX perspective it should only allow one modal at a time to show. Would there be any consideration to disallowing stacking or is that firmly part of the component?

Thanks for the review and help understanding!